cijoe/tests/linkandload/test_xnvme_consumption.py

```diff
@@ -0,0 +1,103 @@
+"""
+This tests whether:
+
+* It is possible to load the shared library
+* It is possible to link with the shared library
+* It is possible to link with the static library
+
+It does so by compiling programs and running them, and additionally, checking
```

Comment:
It looks like it is not actually running the compiled programs. Either that should be implemented or the docstring should be updated

Reply:
Right, now it ought to actually run them....